### PR TITLE
Added compatibilty note for script version of tag

### PR DESCRIPTION
**Pull request by bradmarshall**
Calling cfheader(), the script-based version of this tag, is only available on CF11+. CF10 will trigger a 500 "Variable CFHEADER is undefined" error.

### CODE REVIEW

File: data/en/cfheader.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 	],
 	"engines": {
-		"coldfusion": {"minimum_version":"", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-tags/tags-g-h/cfheader.html"},
+		"coldfusion": {"minimum_version":"", "notes":"Script-based cfheader() added in CF11", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-tags/tags-g-h/cfheader.html"},
```
Comment (shaedrich) [inline data/en/cfheader.json]: You could add a plus sign, so it could be transformed into a label properly: `CF11+` and put it at the beginning of notes.

**Comment by bradmarshall** [comment]
Ah, I didn't realize you could transform those version labels like that. Done! Moved label to front of note as well.

**Comment by shaedrich** [comment]
Yeah, that's pretty nice. So you can have it all at a glance.

**Comment by KamasamaK** [comment]
CFDocs already assumes script syntax is available for tags in ACF11+, Lucee, and Railo 4.2+. If you look at the [CFDocs page for cfheader](https://cfdocs.org/cfheader), you can see that it displays the syntax near the top next to the lightning bolt icon, and if you hover on that icon it will indicate the engines it became available.

**Comment by shaedrich** [comment]
Oh, that didn't even I know.

**Pull request merged**